docs/source/pipeline.rst

```diff
@@ -0,0 +1,314 @@
+               name: str_count
+               params:
+                   pat: white
+           transform_method2:
+               name: add
+               params:
+                   x: 0.5
+       # 7. Using Previously Created Columns
+       col7:
+           sql: col2
+           transform_method:
+               name: add
+               params:
+                   x: 0.5
+
+Train
+~~~~~
+
+For more information on how to customize this step: `Models <https://www.vertica.com/python/documentation/1.0.x/html/machine_learning_vertica.html#api-machine-learning-vertica>`__.
+If you want to train a model, the default setting is to use all the previously created ``cols`` as predictors.
+To subtract the specified columns from the default columns use ``exclude``.
+To strictly choose subset to overide the default columns use ``include``.
+The previous transform example is the basis for this train example:
+
+**Example 1**
+
+.. code:: yaml
+
+     train:    
+       method:
+           name: RandomForestClassifier
+           target: citric_acid
+           params:
+               n_estimators: 40
+               max_depth: 4
+
+**Example 2**
+
+.. code:: yaml
+
+     train:    
+       method:
+           name: LinearSVC
+           target: col2
+           exclude: ['col5', 'col2']
+
+**Example 3**
+
+.. code:: yaml
+
+     train:    
+       method:
+           name: LinearSVC
+           target: col2
+           include: ['col1', 'col3', 'col4', 'col6', 'col7']
+
+Test
+~~~~
+For more information on how to customize this step: `Metric Functions <https://www.vertica.com/python/documentation/1.0.x/html/machine_learning_metrics.html#api-machine-learning-metrics>`__.
+You may want to compute metrics for your newly created model. The results are stored in:
+
+``[schema].[pipeline_name]_METRIC_TABLE``
+
+**Example 1**
+
+.. code:: yaml
+
+     test:
+       metric1: 
+           name: accuracy_score
+           y_true: quality
+           y_score: prediction
+       metric2: 
+           name: r2_score
+           y_true: quality
+           y_score: prediction
+       metric3: 
+           name: max_error
+           y_true: quality
+           y_score: prediction
+
+
+``public.example_METRIC_TABLE``
+
+    .. ipython:: python
+        :suppress:
+
+        import verticapy as vp
+	vdf = vp.vDataFrame(
+		{
+			"metric_name": ['accuracy_score', 'r2_score', 'max_error'],
+			"metric": [0.0, 0.188352265031045, 3.49495733261932],
+		},
+	)
+        html_file = open("SPHINX_DIRECTORY/figures/pipeline_metric_table.html", "w")
+        html_file.write(vdf._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/pipeline_metric_table.html
+
+Scheduler
+~~~~~~~~~
+
+For more information to on how to customize this step: `Cron Wiki <https://en.wikipedia.org/wiki/Cron>`__ and `Vertica Schedulers <https://docs.vertica.com/23.4.x/en/sql-reference/statements/create-statements/create-schedule/>`__.
```

Comment:
My only concern here are the links which should be updated for each new version. Is it not better to add an HTML file in the source of the doc, point to this file which will redirect to the right element so all the references are correct or use a relative path (maybe better as we know in advance the position of the file)? @mail4umar can you help?

Reply:
@mail4umar helped and I added the correct relative paths to VerticaPy methods. I also thought changing the Vertica methods url from: https://docs.vertica.com/23.4.x/en/ -> https://docs.vertica.com/latest/en/ should keep it updated to the newest version.